rollup.config.js

```diff
@@ -1,18 +1,21 @@
-import resolve from 'rollup-plugin-node-resolve';
-import json from '@rollup/plugin-json';
+import resolve from "rollup-plugin-node-resolve";
```

Comment:
Please use single quotes, eslint won't error you but it's currently only configured to run the linting on the `/src` dir (I should change this to run on everything except dist).

Reply:
Okay! Will do.